components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/util/BrandingPreferenceMgtUtils.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.branding.preference.management.core.util;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.wso2.carbon.identity.branding.preference.management.core.constant.BrandingPreferenceMgtConstants;
+import org.wso2.carbon.identity.branding.preference.management.core.exception.BrandingPreferenceMgtClientException;
+import org.wso2.carbon.identity.branding.preference.management.core.exception.BrandingPreferenceMgtServerException;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Util class for branding preference management.
+ */
+public class BrandingPreferenceMgtUtils {
+
+    private static final Log log = LogFactory.getLog(BrandingPreferenceMgtUtils.class);
+
+    /**
+     * Generate Branding Preference input stream.
+     *
+     * @param preferencesJSON JSON string of preferences.
+     * @return Input stream of the preferences JSON string.
+     * @throws UnsupportedEncodingException Unsupported Encoding Exception.
+     */
+    public static InputStream generatePreferenceInputStream(String preferencesJSON) throws
```

Comment:
Do we really need this method?

Reply:
Removed. Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/1/commits/6eb4bd7ef9000f51f02453170ce89aa75ec12ab7